### DISCUSSION TITLE
Why the font displayed on the screen becomes grey (pale)?

### DISCUSSION THREAD

**Discussion by alexeisuv** [brackets-cont/brackets#271] [Q&A]
Why the font displayed on the screen becomes grey (pale)? And how to make it bright again? It actually happened after I pasted some text in.

**Comment by charlypa** [comment]
Could you please share a screenshot?

**Reply by alexeisuv** [reply]
Sure, here it is:
![Screenshot_2023-03-31_at_9_40_58_AM_](https://user-images.githubusercontent.com/61994283/229180534-d3587492-2b93-473b-8d23-54e6b601b9de.png)
